src/services/config/defaults.ts

```diff
@@ -0,0 +1,80 @@
+import fs from "fs-extra";
+import type { Context } from "../command/context.js";
+import { swallowEnoent } from "../filesync/directory.js";
+import { output } from "../output/output.js";
+import { println } from "../output/print.js";
+import { select } from "../output/select.js";
+import { config } from "./config.js";
+
+export type DefaultsConfigData = {
+  telemetry?: boolean;
+  json?: boolean;
+};
+
+export const promptDefaultsConfig = async (ctx: Context): Promise<DefaultsConfigData> => {
+  const path = config.defaultsConfigFile;
+  await fs.ensureFile(path);
+
+  println({ ensureEmptyLineAbove: true, content: "Please answer the prompts to configure ggt defaults." });
+
+  const telemetrySelection = await select({
+    ensureEmptyLineAbove: true,
+    choices: ["enable", "disable"],
+    content: "Would you like to automatically send crash reports and telemetry to Gadget?",
+  });
+
+  const jsonSelection = await select({
+    ensureEmptyLineAbove: true,
+    choices: ["disable", "enable"],
+    content: "Would you like to out as json by default?",
+  });
+
+  const selections: DefaultsConfigData = {
+    telemetry: telemetrySelection === "enable",
+    json: jsonSelection === "enable",
+  };
+
+  fs.writeJSON(path, selections, (err) => {
```

Comment:
Do you mind `await`'ing this `fs.writeJSON` and the one inside of `clearDefaultsConfig` instead of using a callback so that we know the file has finished writing before returning from these functions?

P.S. You can use `fs.outputJSON` instead of `fs.writeJSON` to automatically create the file and parent directories if they don't exist.